现在好像下载不了niconico的会限录播了

[MINYAMI][ERROR] Failed to parse M3U8 file.

一直提示这个

Node.js版本：Node.js v20.11.0
Minyami版本：5.2.0
Minyami（Chrome扩展）：当前最新版
视频地址：https://live.nicovideo.jp/watch/lv344077773

2024年1月20日还能下载，2024年1月21日以后就不能下载了，一直提示[MINYAMI][ERROR] Failed to parse M3U8 file.

下载代码：

minyami -d "https://vodedge793.dmc.nico/hlsarchive/ht2_nicolive/nicolive-production-pg24046263599742_88d787c8498c3f28f22de70b7a9516bc9cc99ab8ac4bdb7b8d0af693cc824143/1/ts/playlist.m3u8?start_time=-572452208373&ht2_nicolive=90567846.wowi2io5n2_s7lmuo_av1juwgo75rg&start=300.357" --output "【シチュボASMR】#ギャルな #魔女 #オイルマッサージ と#ケモ耳女将の#耳舐め - 2024_1_20(土) 23_30開始 - ニコニコ生放送.ts" --key "24046263599742_90567846_1705906030_5cf9487ddfb18d5b8116b0955c8d60aeb9c0673a,super_high"

![20240121144758](https://github.com/Last-Order/Minyami/assets/80573027/52adf0fa-b4cc-4fa1-a78b-2b119b5a37cc)

--- a/package.json
+++ b/package.json
@@ -37,7 +37,8 @@
         "crypto-js": "^4.2.0",
         "erii": "^2.0.6",
         "https-proxy-agent": "^7.0.2",
-        "socks-proxy-agent": "^8.0.2"
+        "socks-proxy-agent": "^8.0.2",
+        "ws": "^7.4.6"
     },
     "devDependencies": {
         "@types/node": "^17.0.17",

